pkgs/servers/http/apache-modules/mod_tile/default.nix

```diff
@@ -1,77 +1,66 @@
-{ lib
+{ fetchFromGitHub
+, lib
 , stdenv
-, fetchFromGitHub
-, fetchpatch
-, autoreconfHook
 , apacheHttpd
 , apr
+, aprutil
+, boost
 , cairo
+, cmake
+, curl
+, glib
+, gnumake
+, gtk2
+, icu
 , iniparser
+, libmemcached
 , mapnik
-, boost
-, icu
-, harfbuzz
-, libjpeg
-, libtiff
-, libwebp
-, proj
-, sqlite
 }:
 
 stdenv.mkDerivation rec {
   pname = "mod_tile";
-  version = "unstable-2017-01-08";
+  version = "0.6.1-master";
```

Comment:
For unreleased packages `nixpkgs` usually uses `unstable-YYYY-MM-DD` format to expose to users material package changes. `master` is a bit non-descriptive. I personally like `0.6.1+unstable=YYYY-MM-DD` format where it's clear which base version we trail and when was the last time snapshot was done.